### ISSUE TITLE
Maintenance of this crate

### ISSUE THREAD

**Issue by carols10cents** [zopfli-rs/zopfli#1]
Hi @kornelski, @shssoichiro, @sagacity, @mqudsi, @dfrankland, @bejolithic, and @AlexTMjugador, 

I've invited you all to be admins on this repo today because you've all authored crates that depend on zopfli or significant contributions to the zopfli crate.

I started the Rust reimplementation as an experiment, and I should have acknowledged a long time ago that I wasn't really up for maintaining it beyond that.

However, you all are depending on this code! So now it's yours and you don't have to wait on me to review your PRs or fix bugs or cut new releases (crates.io invites are coming momentarily).

Please feel free to decide amongst yourselves what to do with this repo. I deliberately created a new repo rather than a fork of https://github.com/carols10cents/zopfli so that I can leave that as the experiment's archive, and this can be the canonical location for future development. I've already updated the URLs on crates.io.

I added a note to this repo's README explaining its origin, it'd be nice if that gets left in there but if you feel like taking it out, I understand :)

Enjoy your new puppy!!!!!! 🐶 🐶 🐶

**Comment by shssoichiro** [comment]
Thanks, Carol! For transparency, for the others mentioned here as well: For my use case, I'm in a happy spot with where this library is. If I have extra time, I might do some performance optimization, but otherwise I won't be touching it much. But I will be available to respond to PRs. 🙂

**Comment by kornelski** [comment]
Thank you

**Comment by AlexTMjugador** [comment]
Thank you! :heart:

I'm also pretty happy with the current state of the library, but I'm glad to help out by keeping an eye on the repository, and maybe submitting patches to upgrade zopfli in other crates. I'm also happy to take responsibility for my PR, in case it is not ergonomic enough for use cases I didn't think about.

**Comment by carols10cents** [comment]
Thank you all!! And I just realized I forgot to invite everyone to be owners of the GitHub org, so you should have that now. Please let me know if there are any other permissions you need from me :)

**Comment by mqudsi** [comment]
Hi Carol. Thanks for the invite. I used this crate for one of my first rust projects to avoid any C dependencies and have never poked around in its source, but I'm happy to help if/where I can. I've pushed some code refactoring the test script (although it might be better to move it over to a proper build system) for the sake of my own sanity to help track down failing tests better. 

The question of how this crate should be published to crates.io remains. I'm not sure if/how you want to give publish permissions and what the standard in the rust/crates.io community has been w/ regards to GitHub organizations and publish rights.

Also, there's a new PR for any maintainers wanting to give it a look: https://github.com/zopfli-rs/zopfli/pull/3

**Comment by mqudsi** [comment]
I'm closing this issue as there's nothing actionable left to be done here. It seems @AlexTMjugador is an owner on crates.io, so my question in the last comment has been answered. Obviously everyone can feel free to continue any maintainer-related discussion in this issue or open a new issue as needed.
